src/prefect/agent/agent.py

```diff
@@ -58,9 +56,27 @@ def start(self) -> None:
         new flow runs to deploy
         """
         tenant_id = self.agent_connect()
+
+        # Loop intervals for query sleep backoff
+        loop_intervals = {
+            0: 0,
```

Comment:
I may have typed this too quickly. This will IMMEDIATELY retry when work is received, which I don't think we should do?